### PR TITLE
Fixes #9647: Update Github Readme for Rudder 4.0 and up

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/9647

### CODE REVIEW

File: README.asciidoc [hunk 1]
```diff
@@ -38,22 +46,54 @@ Screenshots:: https://www.rudder-project.org/site/documentation/screenshots/
...
 ===== Vagrant
 
-Just want to test Rudder without worrying about a full installation?  We provide a https://github.com/Normation/rudder-vagrant[Vagrant set-up]. See https://www.rudder-project.org/site/get-rudder/vagrant/[Rudder Vagrant page] for a getting-started with Rudder on Vagrant.
+Just want to test Rudder without worrying about a full installation?
+We provide a https://github.com/Normation/rudder-vagrant[Vagrant set-up]. See https://www.rudder-project.org/site/get-rudder/vagrant/[Rudder Vagrant page] for a getting-started with Rudder on Vagrant.
+
+==== Rudder Test Framework (rtf)
+
+Rudder test framework is a tool that allows to define full environnement and set
+them up for you. For example, you can define a plateforme with a Rudder server on
```
Comment (amousset) [inline README.asciidoc]: platform

File: README.asciidoc [hunk 2]
```diff
@@ -38,22 +46,54 @@ Screenshots:: https://www.rudder-project.org/site/documentation/screenshots/
...
 
 ===== Vagrant
 
-Just want to test Rudder without worrying about a full installation?  We provide a https://github.com/Normation/rudder-vagrant[Vagrant set-up]. See https://www.rudder-project.org/site/get-rudder/vagrant/[Rudder Vagrant page] for a getting-started with Rudder on Vagrant.
+Just want to test Rudder without worrying about a full installation?
+We provide a https://github.com/Normation/rudder-vagrant[Vagrant set-up]. See https://www.rudder-project.org/site/get-rudder/vagrant/[Rudder Vagrant page] for a getting-started with Rudder on Vagrant.
+
+==== Rudder Test Framework (rtf)
+
+Rudder test framework is a tool that allows to define full environnement and set
```
Comment (amousset) [inline README.asciidoc]: environnements

**Comment by fanf** [comment]
PR rebased

File: README.asciidoc [hunk 3]
```diff
@@ -1,59 +1,108 @@
...
+   or build upon the pre-defined library provided.
 *  Managers can consult compliance reports and access audit logs.
 
-Rudder combines the power of some of the best-of-breed open source tools in the field, adding reporting and higher level features to keep an eye on your IT: http://www.ncf.io/[the ncf framework] and http://cfengine.com/[CFEngine configuration management agent], http://www.fusioninventory.org/[the FusionInventory hardware and software inventory tool], reporting and higher level features to keep an eye on your information system. Rudder’s web interface is developed in Scala. 
-
+Rudder is an established project with several 10000s of node managed, in companies
+from small to biggest-on-their-field. Typical deployement manages 100 to 1000 nodes.
+The biggest known deployment in 2016 is 7000 nodes.
 
 TIP: Rudder is open source – we welcome ideas, comments, criticism, patches and any other contributions! So let's get a chat we our community https://www.rudder-project.org/site/support/chat-mailing-lists/[on irc]!
```
Comment (amousset) [inline README.asciidoc]: This could probably be moved a bit after, after the feature overview for example.

File: README.asciidoc [hunk 4]
```diff
@@ -1,59 +1,108 @@
...
 ===== Linux packages
 
-The easiest way to test and use Rudder is to install it from the provided Linux packages. 
+The easiest way to test and use Rudder is to install it from the provided Linux packages.
 For the server, the main .rpm-based (RedHat, CentOS, SLES) and .deb-based (Debian, Ubuntu)
 distribution are supported. See https://www.rudder-project.org/site/get-rudder/downloads/
-for details about repository and available versions. 
+for details about repository and available versions.
 
 ===== Vagrant
```
Comment (amousset) [inline README.asciidoc]: We may move this just after the demo, and add a comment to explain the differences with the demo (real agent, etc.)

**Comment by fanf** [comment]
PR rebased

File: README.asciidoc [hunk 5]
```diff
@@ -1,122 +1,111 @@
...
-https://github.com/Normation/rudder
+===== Linux server packages
 
-| Rudder inventory | 
-This part is in charge of parsing fusion inventory reports for Rudder
-https://github.com/Normation/ldap-inventory
+The easiest way to test and use Rudder for prototyping or production is to install
+it from the provided Linux packages.
+For the server, the main current .rpm-based (RedHat, CentOS, SLES) and .deb-based
+(Debian, Ubuntu) distribution are supported. See https://www.rudder-project.org/site/get-rudder/downloads/
```
Comment (amousset) [inline README.asciidoc]: distributions

File: README.asciidoc [hunk 6]
```diff
@@ -1,122 +1,111 @@
...
+===== Rudder agent packages
 
-| ncf framework | 
-A powerful and structured CFEngine framework used to build Rudder Techniques
-https://github.com/Normation/ncf
-
-| Rudder tools | 
-Nice tools around Rudder 
-https://github.com/Normation/rudder-tools
+Rudder agent is extremelly versatil and packages are available for the same
```
Comment (amousset) [inline README.asciidoc]: the same on many more versions => a wide set of distributions and releases
(or something like his)

File: README.asciidoc [hunk 7]
```diff
@@ -1,122 +1,111 @@
...
+The easiest way to test and use Rudder for prototyping or production is to install
+it from the provided Linux packages.
+For the server, the main current .rpm-based (RedHat, CentOS, SLES) and .deb-based
+(Debian, Ubuntu) distribution are supported. See https://www.rudder-project.org/site/get-rudder/downloads/
+for details about repository and available versions.
 
-| Rudder agent | 
-This project contains the CLI for Rudder agent
-https://github.com/Normation/rudder-agent
+===== Rudder agent packages
```
Comment (amousset) [inline README.asciidoc]: We should add at the beginning "Rudder agent is packaged for all the platforms supported as servers, and others as well..."

**Comment by fanf** [comment]
PR rebased

File: README.asciidoc (hunk 7 above)
Comment (fanf) [inline README.asciidoc]: I proposed something alike.

**Comment by fanf** [comment]
PR rebased

**Comment by amousset** [comment]
This is way better than before!

**Pull request merged**